crf/src/main/java/org/sagebase/crf/step/active/HeartRateSampleProcessor.kt

```diff
@@ -79,7 +80,7 @@ class HeartRateSampleProcessor @JvmOverloads constructor(val videoProcessorFrame
         // look to see if we have enough to process a bpm
         // Need to keep 2 extra seconds due to filtering lopping off the first 2 seconds of data.
         val meanOrder = meanFilterOrder(roundedRate)
-        val windowLength = (HEART_RATE_WINDOW_IN_SECONDS + 2).roundToInt() * roundedRate + meanOrder
+        val windowLength = (HEART_RATE_WINDOW_IN_SECONDS + 2.0 * HEART_RATE_FILTER_DROP_IN_SECONDS).roundToInt() * roundedRate + meanOrder
```

Comment:
just to check, you weren't just parameterizing this, you also wanted to double the number of seconds you're adding to the window from 2 to 4? (if so, the above comment should be updated to match)

Reply:
Yes, total is 4. It's drop seconds per filter and the low and high filter are done separately.